### PR TITLE
Upgrade parent pom version, update ejb dependency version, and remove…

### PR DESCRIPTION
**Pull request by jim-krueger**
Upgrade pom parent version update ejb dependency version and remove Oracle vendor id

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -310,9 +310,9 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>javax.ejb</groupId>
-            <artifactId>javax.ejb-api</artifactId>
-            <version>3.2-b01</version>
+            <groupId>jakarta.ejb</groupId>
```
Comment (kwsutter) [inline pom.xml]: Is the larger team ready for this change?  I thought that the EJB and JPA artifacts were prematurely published to maven without having a formal Eclipse review...  Eventually, these type of changes need to be done, so maybe we should just start with these and if something changes, we just do another PR...

Comment (jim-krueger) [reply pom.xml]: I agree.  We can start with these and change them if that is needed down the road.

Comment (kwsutter) [reply pom.xml]: +1
